src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -195,6 +196,26 @@ public function getCrawler(): Crawler
         return $this->crawler;
     }
 
+    /**
+     * Returns whether crawler will use masterminds/html5 or not.
+     */
+    public function isHtml5Parser(): bool
+    {
+        return $this->isHtml5Parser;
+    }
+
+    /**
+     * @param bool $isHtml5Parser set usage of masterminds/html5 for crawler.
+     *
+     * @return $this
+     */
+    public function setIsHtml5Parser(bool $isHtml5Parser = true): static
```

Comment:
there is no reason to make the argument optional (especially given that the typical usage will not be with this default value)